app/models/account/states.rb

```diff
@@ -98,6 +100,20 @@ module Account::States
       scheduled_for_deletion.where.has { deleted_at <= value.ago }
     end
 
+    scope :without_traffic_since_time_ago, ->(value = TIME_TO_BE_INACTIVE) do
+      where.has do
+        not_exists Cinstance.where.has { user_account_id == BabySqueel[:accounts].id }.active_since_time_ago(value)
+      end
+    end
+
+    scope :inactive_since_time_ago, ->(value = TIME_TO_BE_INACTIVE) do
```

Comment:
Same as https://github.com/3scale/porta/pull/17/files#r229412297

Reply:
Ok, let's talk about it there 👍